packages/@aws-cdk/region-info/build-tools/aws-entities.ts

```diff
@@ -0,0 +1,37 @@
+/** The names of all (known) AWS regions */
+export const AWS_REGIONS = [
+  'us-east-2',
+  'us-east-1',
+  'us-west-1',
+  'us-west-2',
+  'ap-south-1',
+  'ap-northeast-3',
+  'ap-northeast-2',
+  'ap-southeast-1',
+  'ap-southeast-2',
+  'ap-northeast-1',
+  'ca-central-1',
+  'cn-north-1',
+  'cn-northwest-1',
+  'eu-central-1',
+  'eu-west-1',
+  'eu-west-2',
+  'eu-west-3',
+  'eu-north-1',
+  'sa-east-1',
+].sort();
+
+/** Possibly non-exaustive list of all service names, used to locate service principals */
```

Comment:
explain what happens if a service is not in this list

Reply:
Again, not in this list, no fact is generated. I don't know why I have to explain this... This is the source of elements that are used to generate built-in facts... If it's not in there, it's not built-in.

Reply:
It wasn't clear to me what the responses of this library would be for services that are not listed here. Maybe just add something like "if a service is not listed here, the library will respond with default values based on common AWS patterns"